rosdep/base.yaml

```diff
@@ -3903,7 +3909,6 @@ swi-prolog-clib:
   arch: [swi-prolog]
   debian: [swi-prolog-clib]
   fedora: [pl-devel]
-  gentoo: [dev-lang/swi-prolog]
```

Comment:
I don't think this is reason to remove it, though. I think that just means that it's enabled by default.